scripts/utils/installers/binary-tools.sh

```
@@ -0,0 +1,776 @@
+				rm -rf "$tmpdir"
+				exit 1
+			fi
+			echo -e "${GREEN}✓ Checksum verified${NC}"
+			tar -xzf "$tmpdir/golangci-lint.tar.gz" -C "$tmpdir"
+			cp "$tmpdir/${asset}/golangci-lint" "$BIN_DIR/golangci-lint"
+			chmod +x "$BIN_DIR/golangci-lint"
+			rm -rf "$tmpdir"
+			return 0
+		}
+
+		if [ $DRY_RUN -eq 1 ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)